UnitTests/UnitTests.UWP/UnitTestApp.xaml.cs

```diff
@@ -50,7 +78,10 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
             if (rootFrame == null)
             {
                 // Create a Frame to act as the navigation context and navigate to the first page
-                rootFrame = new Frame();
+                rootFrame = new Frame()
+                {
+                    CacheSize = 0 // Prevent any test pages from being cached
```

Comment:
Is this needed or for performance increases? Also not sure why test page caching would be a bad thing, especially since this function doesn't load a page (like normal UWP apps would do).

Reply:
This frame is only hosting the content we set with our new method. So, we want to ensure it doesn't somehow cache old content for when the next test sets new content. It should probably never happen that way, but I wanted to be explicit about it.

Reply:
Ah I see, got it.